package.json

```diff
@@ -72,10 +74,12 @@
     "lzma": "2.3.2",
     "marked": "0.3.19",
     "mocha": "5.1.0",
+    "plotly.js": "^2.5.0",
```

Comment:
The dependencies this change adds are pretty large -- this one alone is 81.9mb unzipped https://www.npmjs.com/package/plotly.js -- this is notably larger than all the other dependencies we have in pxt combined. I'm not certain the backend would handle things properly / it could cause issues if we tried to create a build with this in it

Reply:
```suggestion
```

@jwunderl are you able to apply this and try again please?

Reply:
Ah, I missed the previous message, sorry! I have a few things on my plate right now that will be a bit hard to shuffle off of at the moment, would it be possible to coordinate with @abchatra on getting help with this?